examples/src/main/java/com/mapbox/navigation/examples/core/BasicNavSdkOnlyActivity.kt

```diff
@@ -0,0 +1,241 @@
+        mapboxMap?.getStyle {
+            val clickPointSource = it.getSourceAs<GeoJsonSource>("CLICK_SOURCE")
+            clickPointSource?.setGeoJson(Point.fromLngLat(latLng.longitude, latLng.latitude))
+        }
+
+        mapboxMap?.locationComponent?.lastKnownLocation?.let { originLocation ->
+            mapboxNavigation?.requestRoutes(
+                    RouteOptions.builder().applyDefaultParams()
+                            .accessToken(Utils.getMapboxAccessToken(applicationContext))
+                            .coordinates(originLocation.toPoint(), null, latLng.toPoint())
+                            .alternatives(true)
+                            .profile(DirectionsCriteria.PROFILE_DRIVING_TRAFFIC)
+                            .build(),
+                    routesReqCallback
+            )
+        }
+        return true
+    }
+
+    private val routesReqCallback = object : RoutesRequestCallback {
+        override fun onRoutesReady(routes: List<DirectionsRoute>) {
+            if (routes.isNotEmpty()) {
+                Snackbar.make(container, String.format(getString(R.string.steps_in_route),
+                        routes[0].legs()?.get(0)?.steps()?.size), LENGTH_SHORT).show()
+
+                // Update a gradient route LineLayer's source with the Maps SDK. This will
+                // visually add/update the line on the map. All of this is being done
+                // directly with Maps SDK code and NOT the Navigation UI SDK.
+                mapboxMap?.getStyle {
+                    val clickPointSource = it.getSourceAs<GeoJsonSource>("ROUTE_LINE_SOURCE_ID")
+
+                    val routeLineString = LineString.fromPolyline(routes[0].geometry()!!,
+                            6)
+                    clickPointSource?.setGeoJson(routeLineString)
+                }
+                route_retrieval_progress_spinner.visibility = INVISIBLE
+            } else {
+                Snackbar.make(container, R.string.no_routes, LENGTH_SHORT).show()
+            }
+        }
+
+        override fun onRoutesRequestFailure(throwable: Throwable, routeOptions: RouteOptions) {
+            Timber.e("route request failure %s", throwable.toString())
+            Snackbar.make(container, R.string.route_request_failed, LENGTH_SHORT).show()
+        }
+
+        override fun onRoutesRequestCanceled(routeOptions: RouteOptions) {
+            Timber.d("route request canceled")
+        }
+    }
+
+    /**
+     * Enable the Maps SDK's LocationComponent
+     */
+    private fun enableLocationComponent() {
+        mapboxMap?.getStyle {
+            val locationComponentActivationOptions =
+                    LocationComponentActivationOptions.builder(this, it)
+                            .build()
+
+            // Get an instance of the LocationComponent and then adjust its settings
+            mapboxMap?.locationComponent?.apply {
+
+                // Activate the LocationComponent with options
```

Comment:
Do we need this comment? I think method name is self-explanatory. Same applies to comments below 👇

Reply:
Not needed. Removed the comments.